src/public/styles.css

```diff
@@ -154,3 +168,38 @@ nav > ul {
   font-weight: lighter;
   margin-top: 30px;
 }
+
+.password-ul {
+  /* border: 2px solid red; */
+  margin-left: -50px;
+  margin-top: 0px;
+}
+
+ul li {
+  list-style-type: none;
+}
+
+i {
```

Comment:
I think this is too vague, it adds 0.5rem to every <i> across the whole site. Could you have a selector which only targets <i> within a specified container, or you could you add a class/use an existing class?